boa_engine/src/syntax/ast/node/mod.rs

```diff
@@ -348,6 +349,92 @@ where
     buf
 }
 
+/// `FormalParameterList` is a list of `FormalParameter`s that describes the parameters of a function.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///
+/// [spec]: https://tc39.es/ecma262/#prod-FormalParameterList
+#[cfg_attr(feature = "deser", derive(Serialize, Deserialize))]
+#[derive(Clone, Debug, Default, PartialEq, Trace, Finalize)]
+pub struct FormalParameterList {
```

Comment:
Maybe we could move this to its own submodule to reduce the size of this file. We should probably do this with many other things, but we should not add more stuff to the file.

Then, just for my curiosity, did you check that we no longer use `[FormalParameter]` or `Vec<FormalParameter>` anywhere?

Reply:
I moved the parameter code to it's own file and also moved the property definition code to the object ast node, as is is used there.

Good catch with `[FormalParameter]`, there was one usage left.